index.ts

```diff
@@ -124,4 +100,31 @@ const xmlQuery = (ast: XmlNode|XmlNode[]) => {
     };
 };
 
+namespace xmlQuery {
+
+    export interface XmlNode {
```

Comment:
I had to do this because these types were used in the xmlQuery input, so need to be exported too.
And there are 2 ways to achieve this:
- Use ES6 style of export: export interfaces, and export default xmlQuery.
- Export only the xmlQuery to conform the CommonJS style, but "attaching" the interface types to it, so those will be exported too under a "namespace" (with any JS effect, only for TS).